### DISCUSSION TITLE
Is there a timeline to support spark 3.2 ?

### DISCUSSION THREAD

**Discussion by RamakrishnaChilaka** [jaceklaskowski/mastering-spark-sql-book#8] [General]

**Comment by jaceklaskowski** [comment]
Not really a timeline, but a wish for sure. Given other commitments and interesting OSS projects I'm focusing on (currently [ksqlDB](https://github.com/japila-books/ksqldb-internals)) I can't promise much.

If we end up with small bites though I could have them done earlier. What would be the most interesting topic to read about first?

**Reply by RamakrishnaChilaka** [reply]
@jaceklaskowski , I was planning to go through new logical plan changes..in the new updated document.

**Reply by jaceklaskowski** [reply]
I'll have a look at the changes this week...
